cmd/kubeadm/app/phases/upgrade/postupgrade_test.go

```diff
@@ -19,12 +19,17 @@ package upgrade
 import (
 	"os"
 	"path/filepath"
+	"reflect"
 	"regexp"
 	"strings"
 	"testing"
 
 	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes/fake"
 
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
The best order to import is like this, with blank lines between each section:

stdlib
external
external k8s.io
kubeadm

Reply:
good catch. thanks.